pytest_tap/tests/test_plugin.py

```diff
@@ -135,4 +135,4 @@ def test_path_pytest(self):
         report = mock.Mock(when='call', outcome='passed', location=location)
         plugin.pytest_runtest_logreport(report)
         plugin.tracker.add_ok.assert_called_once_with(
-                'tests/test_file.py', 'TestFake.test_me')
+                'tests/test_file.py', 'tests/test_file.py::TestFake.test_me')
```

Comment:
Looks like there are extra spaces here. Could you update this so the indentation is only 4 spaces?